api/controller/helpers.go

```
@@ -0,0 +1,19 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func responseHandle(w http.ResponseWriter, data interface{}, status int) {
+	w.Header().Add("Content-Type", "application/json")
+	w.WriteHeader(status)
+
+	if data == nil {
+
+		return
+	}
+	js, _ := json.Marshal(data)
```

Never ignore errors; an error might lead to stopping the current program execution, which is not expected to happen.

You should handle the error properly.